documentation/asciidoc/computers/config_txt/common.adoc

```diff
@@ -0,0 +1,39 @@
+== Common Options
+
+=== Common Display Options
+
+==== `disable_overscan`
+
+Set `disable_overscan` to `1` to disable the default values of xref:configuration.adoc#underscan[overscan] that is set by the firmware. The default value of overscan for the left, right, top, and bottom edges is `48` for HD CEA modes, `32` for SD CEA modes, and `0` for DMT modes. The default value for `disable_overscan` is `0`.
+
+==== `hdmi_enable_4kp60` (Raspberry Pi 4 Only)
+
+By default, when connected to a 4K monitor, the Raspberry Pi 4B, 400 and CM4 will select a 30hz refresh rate. Use this option to allow selection of 60Hz refresh rates. Note, this will increase power consumption and increase the temperature of the Raspberry Pi. It is not possible to output 4Kp60 on both micro HDMI ports simultaneously.
+
+=== Common Hardware Configuration Options
+
+==== `auto_detect_cameras`
+
+With this setting enabled (set to `1`), the firmware will automatically load overlays for cameras that it recognises. New Rasperry Pi OS images from Bullseye onwards come with this setting by default.
+
+==== `auto_detect_displays`
+
+With this setting enabled (set to `1`), the firmware will automatically load overlays for displays that it recognises. New RPiOS images from Bullseye onwards come with this setting by default.
```

Comment:
Same here with the "New RPiOS images from Bullseye onwards come with this setting by default."